src/cloud-api-adaptor/pkg/podnetwork/tunneler/vxlan/podnode.go

```diff
@@ -25,6 +24,7 @@ func NewPodNodeTunneler() tunneler.Tunneler {
 
 func (t *podNodeTunneler) Setup(nsPath string, podNodeIPs []netip.Addr, config *tunneler.Config) error {
 
+	podVxlanInterface := config.InterfaceName
```

Comment:
Same here.

Reply:
We need an null string check here as well.

I noticed another issue here. podVxlanInterface is created in the host network interface first, and then it is move to the pod network namespace at Line 61 `vxlan.SetNamespace(podNS)`.

If podVxlanInterface is some common network interface name like eth0, then it may conflict with interfaces in the host network namespace. We may use a fixed non-common interface name for vxlan interface, and then we can rename it after it is moved to the pod network namespace.

I'll fix this issue.

Reply:
done